content/docs/user-guide/data-management/index.md

```diff
@@ -0,0 +1,168 @@
+and repeat going forward (by anyone). There's no need to come up with naming
+schemes for changed data and model files.
+
+![Versioning data with Git](/img/project_versioning.png) _You can use Git
+history to store different dataset and model versions without renaming any files
+in your workspace. The project cache grows as more relevant versions are
+tracked._
+
+## How it works
+
+Let's consider a simplified, hypothetical ML project:
+
+```
+training.csv
+validation.xml
+model.bin
+src/train.py
+```
+
+DVC appends unique <abbr>project</abbr> data and model file contents to a hidden
+<abbr>cache</abbr>. Entries in this data store are organized by content hashes,
+similar to an index.
+
+```cli
+.dvc/cache
+├── 0a/aa77e # training.csv
+├── 3f/db533 # validation.xml before
+├── 6a/2aa4b # validation.xml now
+├── a7/28107 # first model.bin
+    ...
+```
+
+<!-- But what about old versions? -->
+
+<!-- While DVC caching prevents file duplicates (optimizing storage), it can also store old data that's no longer relevant. -->
+
+<!-- Prevents accidental deletions. -->
+
+Now that they're cached safely, DVC-tracked data and models in the
+<abbr>workspace</abbr> can be [replaced with file links], so you can continue
+seeing and using them. File hashes (usually MD5) are written in human-readable
+YAML [metafiles] next to the original data.
+
+```git
+  training.csv -> .dvc/cache/0a/aa77e
++ training.csv.dvc
+  validation.xml -> .dvc/cache/6a/2aa4b
++ validation.xml.dvc
+  model.bin
+  ...
+```
+
+```yaml
+# validation.xml.dvc
+  ...
+  md5: 6a26845d4000daa4bfb196017e103355
+  path: validation.xml
+```
+
+[metafiles]: /doc/user-guide/project-structure
+[replaced with file links]:
+  /doc/user-guide/data-management/large-dataset-optimization
+
+<admon type="info" title="Data codification">
+
+DVC replaces data in the project with code-like files (and links) as explain
+above. Codifying data lets you treat it as a first-class citizen in any code
+repository because metafiles can be [versioned with Git] (or any SCM) along with
+code, config, etc.
+
+[versioned with git]:
+  https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control
+
+</admon>
+
+<!-- Technical diagram? -->
+
+<!-- It's also cleaner since there's no need for special file names like `2020-03_training_split (256x256)` to organize variations (use repo branches or tags instead). -->
+
+## Benefits and tradeoffs
+
+Adopting DVC's approach requires a few key changes to your workflow:
+
+1. Relevant data and models are registered in a code repository (typically Git).
+1. Data operations (add, remove, move, etc.) happen [indirectly]: DVC checks the
+   metadata to locate files in both sides.
+1. Stored objects managed with DVC are not intended for handling manually.
+
+[indirectly]: https://en.wikipedia.org/wiki/Indirection
+
+At the same time, it comes with many benefits (recap):
+
+- Easily manage **data as code** and [optimize space usage][file links]
```

Comment:
minor: broken link

Reply:
Here and everywhere, review carefully these items:

`Easily manage **data as code** and [optimize space usage][file links]` 

- two benefits at once?
- one is very abstract - `data as code` - it won't resonate
- second is about optimization - fine, but it comes first in the list - is it the main thing?
- I would need to follow the link to understand the benefit?

do we need the whole benefits section? may be How it works can be enough as it describes how you change your workflow ... ? 

not sure about the structure here tbh

Reply:
I hid the benefits list for now.